src/Core/Format.php

```diff
@@ -52,16 +42,17 @@ public function __construct($xml)
     /**
      * @return Image[]
      */
-    public function getImages()
+    public function getImages(): array
```

Comment:
Trade-off between more memory (old version) and more cpu (new version). Need a good argument to decided which solution to choose for the community. We assume the XML content is static and never changes in a single API call.